integrations/mysql/src/mysql.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	integrationName    = "com.newrelic.mysql"
-	integrationVersion = "1.0.0"
+	integrationVersion = "1.0.1"
```

Comment:
I wouldn't consider all these changes as a "fix", so probably would make sense to increase the minor number.